### DISCUSSION TITLE
Can I integrate turso with PocketBaseCMS

### DISCUSSION THREAD

**Discussion by hackemateninja** [pocketbase/pocketbase#5147] [Q&A] [answered]
I am thinking to start using pocket base, for my project but I would like to know if the is possible to integrate turso with pocketbase CMS

**Comment by ganigeorgiev** [comment, marked as answer]
At the moment No (at least not out of the box).

For more details, please check https://github.com/pocketbase/pocketbase/issues/3032#issuecomment-1959278479.

If your use case requires horizontal scaling, then PocketBase is not suitable for it.

In any case, for me the only valid use case for using PocketBase with something like Turso is autosync/replication for disaster recovery (which you could already achieve with Litestream).
If you think that you'll get somehow better performance by integrating PocketBase with Turso (_or any other remote database_) you are probably wrong as the latency of the network roundtrips will be almost always bigger than what is essentially a inprocess function call.
